Automattic/vip-design-system
# Replace Toggle component using Radix

## Description

Replacing our current own implementation of Toggle, with the Radix Switch component. The Radix version has A11Y improved, and you can even use a custom label without violating any standard.

In this PR, I also added the controls addons to have better documentation in the storybook and play around with the components.

https://www.radix-ui.com/docs/primitives/components/switch

## Checklist


## Steps to Test

1. Open https://deploy-preview-75--vip-design-system-components.netlify.app/?path=/story/toggle--external-label
2. Switch the toggle with the label and click
3. Check that no violations for A11Y were found

![image](https://user-images.githubusercontent.com/3402/177415478-86220673-bdfd-4104-a429-8f10ed06cd8b.png)
